PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,37 @@
+---
 <!-- Thanks for contributing to 'routing-release'. To speed up the process of reviewing your pull request please provide us with: -->
 
-* A short explanation of the proposed change:
+# What is this change about?
 
-* An explanation of the use cases your change solves
+_In your own words, describe this proposed change and the problem it solves._
 
-* Instructions to functionally test the behavior change using operator interfaces (BOSH manifest, logs, curl, and metrics)
+# What type of change is this?
 
-* Expected result after the change
+- [ ] `[Breaking Change]`: the change removes a feature or introduces a behavior change to core functionality (request routing, request logging)
+- [ ] `[Minor Feature/Improvement]`:  the change introduces a new feature or improvement that doesn't alter core behavior
+- [ ] `[Bug Fix]`: the change addresses a defect
 
-* Current result before the change
+_If you have selected multiple of the above, it may be wise to consider splitting this PR into multiple PRs to decrease the time for minor changes + bugs to be resolved._
 
-* Links to any other associated PRs
+# Backwards Compatibility
 
-* [ ] I have viewed signed and have submitted the Contributor License Agreement
+_If this is a breaking change, or modifies currently expected behaviors of core functionality (request routing or request logging), how has the change been mitigated to be backwards compatible?_
 
-* [ ] I have made this pull request to the `develop` branch
+_Should this feature be considered experimental for a period of time, and allow operators to opt-in? Should this apply immediately to all routing-release deployments?_
 
-* [ ] I have run all the unit tests using `scripts/run-unit-tests-in-docker`
+# How should this be tested?
 
-* [ ] (Optional) I have run Routing Acceptance Tests and Routing Smoke Tests on bosh lite
+_Are there any non-automated tests that should be performed against this change for validation? Please provide steps, and expected results for before + after the change._
+
+# Additional Context
+
+_Please provide any additional links or context (issues, other PRs, Slack discussions) to help understand this change._
 
+# PR Checklist
+* [ ] I have viewed signed and have submitted the Contributor License Agreement
+* [ ] I have made this pull request to the `develop` branch
+* [ ] I have run all the unit tests using `scripts/run-unit-tests-in-docker`
+* [ ] I have given thought to the backward-compatibility requirements of this change, and listed any mitigations above.
+* [ ] (Optional) I have run Routing Acceptance Tests and Routing Smoke Tests on bosh lite
```

Comment:
Not sure whether bosh lites specifically are a good barometer anymore as afaik they aren't widely used. Perhaps the "on bosh lite" part of this and the next bullet should be cut?

Reply:
Agreed.

Reply:
Done